sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -1,11 +1,15 @@
 # Azure Text Analytics client library for Java
```

Comment:
The links for samples are broken. https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/textanalytics/azure-ai-textanalytics/src/samples#azure-text-analytics-client-library-for-java-samples
Could you also fix those in this PR?